ext/bg/settings.html

```diff
@@ -134,6 +134,10 @@ <h3>General Options</h3>
                     <label><input type="checkbox" id="enable"> Enable content scanning</label>
                 </div>
 
+                <div class="checkbox">
+                    <label><input type="checkbox" id="enable-clipboard-popups"> Enable native popups when copying Japanese text</label>
```

Comment:
This option should be hidden for Android, or at least disabled.

If you want a multi-purpose solution, you could do something similar to the existing CSS rules `data-show-for-browser="firefox-mobile"`, except like `data-hide-for-browser`.

Reply:
Added `data-hide-for-browser`, but noticed that it's visible for a while before the settings page script has run. Shouldn't be a big issue